### ISSUE TITLE
URS: Upgrade information for ALL units able to upgrade

### ISSUE THREAD

**Issue by JHCD** [CQUI-Org/cqui#492]
For a short time the upgrade information is broken in the UnitReportScreen, when an unit is abroad.

![20170410205817_1](https://cloud.githubusercontent.com/assets/12496340/24877855/1bcbfa04-1e31-11e7-8c25-1086b5c2f2b9.jpg)

![20170410205805_1](https://cloud.githubusercontent.com/assets/12496340/24877835/06583d72-1e31-11e7-8efb-2e2b50b1bc85.jpg)

**Comment by ricanuck** [comment]
What?

**Comment by ricanuck** [comment]
You cannot upgrade "abroad".

**Comment by chaorace** [comment]
Perhaps he means "Embarked". That seems like the kind of thing that can be lost in translation

**Comment by JHCD** [comment]
Sorry, incomplete text:

In the past, there was the same "disabled upgrade arrow" in the URS like in this unit panel.
At the moment there is no hint in the URS, so you have to click on every unit ouside your country and check if it is able to upgrade. Especially when all units from the same typ are abroad.

**Comment by ricanuck** [comment]
save please.

**Comment by ricanuck** [comment]
OK, looked at the code in URS. I think your memory is playing you some funny tricks, because: 1) I don't remember ever seen the disabled upgrade arrows in URS, and 2) the code clearly does not include showing the disabled arrow. The idea of that page of URS is to only give you the ability to upgrade quickly from that page, nothing else.

With that said, we could include more code to show the disabled arrows. But that would be an enhancement, not a bug.

(Point being, I don't like seeing a lot of "BUG" red banners when entering the Issues page... think of someone that is a first visitor coming to check the status of the mod... add to that the still prevalent opinion that "CQUI is good but full of bugs"... and you know what I mean. We should be careful with the "BUG" flag, or else...)

Bottom line, this is not a BUG.

**Comment by ricanuck** [comment]
OK, I have it working like the Unit Panel now, working fine with my save, but if you want to test it, here is the file:
https://drive.google.com/open?id=0B_TA2f5G0Z78b215b1d0c253REU

**Comment by JHCD** [comment]
Please stop calling me an idiot... I'm not your 10 years old son...
That is the third time here...

**Comment by ricanuck** [comment]
What?

**Comment by JHCD** [comment]
Thanks, looks like before one of the CIV patches...
